nrf_wifi/fw_if/umac_if/inc/fw/patch_info.h

```diff
@@ -58,6 +58,6 @@ struct nrf70_fw_image_info {
 #define RPU_FAMILY         (1)
 #define RPU_MAJOR_VERSION   (2)
 #define RPU_MINOR_VERSION   (10)
-#define RPU_PATCH_VERSION   (25)
+#define RPU_PATCH_VERSION   (27)
```

Comment:
Will this be version#26,

Reply:
This will be 27 itself. I have corrected it now.